topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+>   - Is metadata for the resource provided in both human- and machine-readable format (e.g JSON, XMLor YAML)?
+> - **Accessible**
+>   - Has an accessibility checker tool been utilised to improve the accessibility of all learning resource files (PDF, HTML, video, etc.)?
+>   - Are access rules (authentication & authorisation) implemented for the learning resource?
+> - **Interoperable**
+>   - Is the RDA minimal (or domain specific) metadata schema used for the learning material description?
+>   - Is the resource available in open file formats which are tool agnostic and compatible with a wide variety of existing software?
+> - **Reusable**
+>   - Is there clear attribution for all reused resources with compatible licenses?
+>   - Has the learning resource been made available for use by defining a permissable license or policy information that allows derivations?
+{: .details}
+
+><details-title>Optional requirements</details-title>
+> - Did you follow the stages of the backward instructional design process while developing the learning resource?
+> - Are controlled vocabularies (CVs) used for describing the resource characteristics aligned with the chosen metadata schema?
+> - Does the learning resource represent a complete learning object defined around minimum one learning objective?
+> - Does the resource incorporate an instructor kit that aids in facilitating the process of others reusing learning material by offering helpful how-to guides?
+>    - facilitator guide
+>    - activities description
+>    - assessment activities and strategy to assess
+>    - general learning content or instructor notes
+>    - lesson unit plan
+>    - syllabus
+> - Have you employed a versioning system to track and control changes in your materials?
+> - Are the resource access rules (how to access, e.g. registration procedure) explicitly communicated to learners?
+> - Is the learning resource searchable in at least one relevant catalogue?
+>    - Is it FAIR (can be searched based on metadata)?
+> - Does the course include the possibility to provide feedback or comments from users and-or trainers/designers?
+>    - If so, do you regularly gather and analyse that feedback?
+> - Does the resource adopt an open community approach regarding its quality and reachability?
+> - Has the learning resource been checked by a third party regarding its learning experience quality?
+{: .details}
+
+### Feedback QA
+
+> <comment-title>Regularly gather feedback from learners and instructors</comment-title>
+> Ensure that you actively and regularly gather feedback from both perspectives: the learners and the instructors.
+{: .comment}
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/416-verify/" class="btn btn-dark text-white btn-lg btn-block">Go to the full FAIR-by-Design Methodology: Verify stage ...</a>
```

Comment:
In the Galaxy/GTN world, a full-width button like this would would imply an expandable section to our users, rather than a link that takes them to a different page, so perhaps we adjust the styling of this a bit to make it more clear this is a link that will lead them to a different page

Reply:
We changed all the buttons from full-width to stretchable button depending on the text within.